md/pages-development.md

```diff
@@ -0,0 +1,143 @@
+# Develop a page
+
+A page presents information to a user or can also collect information entered by a user.  
+Displayed in a browser, a [page](pages.md) is intended for use in applications. 
+
+## Provided pages
+You do not need to create all your pages from scratch. Bonita already provides some pages.  
+Bonita Portal "Ressources" section lists all the non-UI Designer-made pages, that can be reused as is in an application.
+By downloading the Bonita User and Administrator applications from Bonita Studio Welcome page, you also download Bonita pages created with Bonita UI Designer.  
+Those pages can be customized and added to an application.
+
+## Custom pages
+A page resource has the general [resource definition](resource-management.md). 
+If it contains an `Index.groovy` file, this must implement the PageController interface optionally with libraries. 
+If you create a custom page with Bonita UI Designer, its required structure and content are right, by default.
+
+A custom page is displayed inside an iframe to prevent conflicts between Bonita Portal resources (for example JS and CSS) and those used in the custom page. 
+This also reduces the risk of migration issues, for example if a custom page uses the version of JQuery provided with Bonita Portal and it is updated.
+
+For all information about developping a custom page, go to the dedicated page
```

Comment:
Isn't it this page ? if there is another one, can we put a link ?

Reply:
sure :)